README.md

```diff
@@ -265,33 +266,31 @@ class Vikingo {
   // para qué un constructor con los parámetros
   // 1. me aseguro que construir un objeto sea una operación atómica (no queda en
   // estado inconsistente)
-  // 2. quiero que el vikingo sea inmutable
-  constructor(props) {
+  // 2. Tomo sólo las propiedades que me interesan, de otra manera podrian sobreescribirme los slots correspondientes a métodos!
```

Comment:
El `Object.assign` no produce una copia ni hace que sea inmutable.
De hecho, el `new` produce una copia.
Los pasos del `new` son...

- crear un objeto nuevo `{}` que tiene asignado `__proto__` al objeto definido en la clase
- llamar al `constructor` con ese objeto como contexto `this`, algo así como `{}.constructor()`
- si la función `constructor` retorna otra cosa, arroja eso como resultado del `new` sin importar nada

Reply:
Uh, quedó el comentario de la versión inmutable, esta versión nada que ver con la inmutabilidad. Hay que borrar ese comment, está perfecto.

Mi confusión del `Object.assign` parte porque se lo suele utilizar como una forma de shallow copy. Así que +1 a tu comentario.